### ISSUE TITLE
eventlet module

### ISSUE THREAD

**Issue by wc1018** [spyder-ide/spyder#21015]
### What steps will reproduce the problem?

import time
import eventlet  #导入eventlet这个模块
eventlet.monkey_patch()   #必须加这条代码
with eventlet.Timeout(1e-5,False):   #设置超时时间为2秒
   print ('这条语句正常执行')
   time.sleep(10)
   print ('没有跳过这条输出')
print ('跳过了输出')

### Traceback
```python-traceback
  File "D:\anaconda3\lib\site-packages\zmq\eventloop\zmqstream.py", line 444, in _run_callback
    callback(*args, **kwargs)
  File "D:\anaconda3\lib\site-packages\jupyter_client\threaded.py", line 91, in _handle_recv
    ident,smsg = self.session.feed_identities(msg)
  File "D:\anaconda3\lib\site-packages\jupyter_client\session.py", line 850, in feed_identities
    idx = msg_list.index(DELIM)
ValueError: b'<IDS|MSG>' is not in list
ERROR:tornado.application:Exception in callback functools.partial(<bound method ThreadedZMQSocketChannel._flush of <qtconsole.client.QtZMQSocketChannel object at 0x1503EAD8>>)
Traceback (most recent call last):
  File "D:\anaconda3\lib\site-packages\tornado\ioloop.py", line 741, in _run_callback
    ret = callback()
  File "D:\anaconda3\lib\site-packages\jupyter_client\threaded.py", line 143, in _flush
    self.stream.flush()
  File "D:\anaconda3\lib\site-packages\zmq\eventloop\zmqstream.py", line 364, in flush
    self._handle_recv()
  File "D:\anaconda3\lib\site-packages\zmq\eventloop\zmqstream.py", line 492, in _handle_recv
    self._run_callback(callback, msg)
  File "D:\anaconda3\lib\site-packages\zmq\eventloop\zmqstream.py", line 444, in _run_callback
    callback(*args, **kwargs)
  File "D:\anaconda3\lib\site-packages\jupyter_client\threaded.py", line 91, in _handle_recv
    ident,smsg = self.session.feed_identities(msg)
  File "D:\anaconda3\lib\site-packages\jupyter_client\session.py", line 850, in feed_identities
    idx = msg_list.index(DELIM)
ValueError: b'<IDS|MSG>' is not in list
WARNING:traitlets:No such comm: 7efc8e3107f311ee8e9bf077c31c5d03
WARNING:traitlets:No such comm: 7efc8e3107f311ee8e9bf077c31c5d03
```

## Versions

* Spyder version: 4.2.5 
* Python version: 3.8.8
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  7.22.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  22.3.0 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.8.0 (OK)
pygments >=2.0                  :  2.8.1 (OK)
pylint >=1.0                    :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.2             :  0.3.2 (OK)
qdarkstyle >=2.8;<3.0           :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.2 (OK)
qtconsole >=5.0.3               :  5.0.3 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.7 (OK)
setuptools >=39.0.0             :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                  :  4.0.1 (OK)
spyder_kernels >=1.10.2;<1.11.0 :  1.10.2 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0        :  1.0.2 (OK)
zmq >=17                        :  19.0.2 (OK)

# Optional:
cython >=0.21                   :  0.29.23 (OK)
matplotlib >=2.0.0              :  3.3.4 (OK)
numpy >=1.7                     :  1.24.3 (OK)
pandas >=1.1.1                  :  1.2.4 (OK)
scipy >=0.17.0                  :  1.6.2 (OK)
sympy >=0.7.3                   :  1.8 (OK)
```

**Comment by ccordoba12** [comment]
Hey @wc1018, thanks for reporting. This problem should be fixed in our latest version available in Anaconda (5.4.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
